lib/config/rules/jest.js

```diff
@@ -0,0 +1,20 @@
+// see https://github.com/sindresorhus/eslint-plugin-ava
+
+module.exports = {
+  // Disallow all forms of skipped tests (`.skip`, prepending `x` to test helpers, empty function bodies, `pending()` calls.)
+  'jest/no-disabled-tests': 'error',
+  // Disallow disabling tests via appending `.only` or prepending `f` to test helpers.
+  'jest/no-focused-tests': 'error',
+  // Make it easier to find failing tests by preventing duplicate test names in a suite.
+  'jest/no-identical-title': 'error',
```

Comment:
I can see how tests in different `describe` blocks can reasonably have the same title.  However, the rule flagged at least two duplicate tests in `shopify/web`, and it was easy enough to make other clashes distinct.
  
-----

[Deleted duplicate test 1](https://github.com/Shopify/web/pull/2337/files#diff-d88afea9466e759ef531a84612b2ef2cL260)
[Deleted duplicate test 2](https://github.com/Shopify/web/pull/2337/files#diff-700280fbd3af1a10bd3f9b02c27a703fL12)
[Possibly another duplicate?](https://github.com/Shopify/web/pull/2337/files#diff-48e07a6d8403126100ec743d3c8e5eb1R485)

[Renamed test 1](https://github.com/Shopify/web/pull/2337/files#diff-56bcd9f065ecc470eb9bb3577a898086R416)
[Renamed test 2](https://github.com/Shopify/web/pull/2337/files#diff-56bcd9f065ecc470eb9bb3577a898086R487)